### PR TITLE
Python install docs

### PR DESCRIPTION
**Pull request by rabernat**
@kerrykey: This is what it would look like if we made our updates via pull requests

### CODE REVIEW

**Comment by rabernat** [comment]
You could review what I wrote and we could discuss it / modify it before deciding to merge.

Overkill for this simple website, but just providing an example.

One case where this *would* be useful is if I wanted to propose a change to one of your lectures. I  wouldn't want to just commit it without getting your approval first. So a PR would be appropriate.

**Comment by rabernat** [comment]
Feel free to click merge! I'm going to bed. ;)

**Review by kerrykey — APPROVED** [review]
looks good.

**Pull request merged**
